pyrobosim/examples/demo.py

```diff
@@ -119,7 +119,7 @@ def create_world(multirobot=False):
         from pyrobosim.navigation.rrt import RRTPlanner
```

Comment:
Should `from pyrobosim.navigation.rrt import RRTPlanner` import go at the top with other imports?

Reply:
Yes! Thank you